docs/UserGuide.md

```diff
@@ -21,250 +28,390 @@ commands, you can have complete control of your perishable goods.
+- ❗ : warning
 
-Command: `[item] delete ITEM_INDEX`
+If you are stuck, refer to [Troubleshooting](#troubleshooting) or [FAQ](#faq).
+There is also a [Glossary](#glossary) that contains definitions of what common words 
+used in this application mean.
 
-> Description: Deletes a specified item. Returns a warning if the item does not exist.
+It is time for you to unleash the potential of a command line application!
 
----
+## Quick Start
 
-Example:
+Hello! This is the command box where we key in our commands.
 
-Input
+{Image of a command box}
 
-```
-delete 1
-```
+([Navigating around the application](#navigating-around-the-application))
 
-Output
+The format for commands are not identical. One command in FoodRem is the command to create a new item.
 
-```
-(Item exists): Item “potato” successfully deleted!
-(Item does not exist): No item to be found at index 1. Use “list items” or “find NAME” to find the index of the item to be deleted.
-```
+Format: `new n/ITEM_NAME [qty/QUANTITY] [type/TYPE] [bgt/BOUGHT_DATE] [exp/EXPIRY_DATE]`
 
-#### Item Update
+The first word of every command allows FoodRem to distinguish different commands.
+`new` tells FoodRem that this is the command to create a new item.
+[Flags](#flags) such as `n/` and `qty/` are delimiters that enable FoodRem to distinguish different parameters
+supplied by you without ambiguity. [Placeholders](#placeholders) such as `ITEM_NAME` and `QUANTITY` shows you
+what you should place in each portion of the command. 
 
-Command: `item set ITEM_NAME [ARGS]`
+Suppose you just bought 30 kg of potatoes, today is 5th September 22, and you do not feel the need to record
+an expiry date for the potatoes.
 
-> Description: Updates properties of an item.
+`ITEM_NAME`: Potatoes
 
----
+`QUANTITY`: 30
 
-Example:
+`TYPE`: kg
 
-Input
+`BOUGHT_DATE`: 05-09-22
 
-```
-item set ARGS
-```
+Note: 
+- The [Placeholder](#placeholders) section covers the restrictions for respective placeholders.
+For example, the date format of BOUGHT_DATE, certain characters you cannot use and the limit and precision of numbers.
 
-Output
+The command you would like to enter into the command box would be:
 
-```
-Item successfully updated
-```
+`new n/Potatoes qty/30 type/kg bgt/05-09-22`
 
-## Commands
+Alternatively these commands would do the same thing:
+- `new n/Potatoesqty/30type/kgbgt/05-09-22` (Omitting space between tags)
+- `new qty/30 n/Potatoes bgt/05-09-22 type/kg` (Reordering the flags)
 
-### Tags
+These commands are invalid:
+- `newn/Potatoesqty/30type/kgbgt/05-09-22` (Removing space between command identifier and flag)
+- `new qty/-48 n/PÖtátÖes bgt/05/09/22 type/|kg|` (Restrictions of placeholders not followed)
 
-#### Tag Create
+Find out more about restrictions in the sections [Flags](#flags), [Placeholders](#placeholders) and 
+[Features](#features).
 
-Command: `tag create TAG_NAME`
+You might also realise that there is a pair of square brackets `[]` surrounding some parameters like `qty/QUANTITY` 
```

Comment:
This would sound better:
Notice that there is a pair of square brackets `[]` surrounding some parameters like...

Reply:
Note, will make the necessary changes.